Specs/Scene/LabelCollectionSpec.js

```diff
@@ -875,16 +948,23 @@ defineSuite([
                     continue;
                 }
 
-                width += billboard.width;
-                height = Math.max(height, billboard.height);
+                var glyphWidth = billboard.width * scale;
```

Comment:
Yeah, to be honest I'm not happy with this repeated code being here at all.  Different test platforms have different font metrics, so the "correct" answers need to be calculated inside the test.  As a result these tests (in master) have a copy-and-paste of part of the body of the function they are testing in the test itself.  I'm no testing expert but I'm sure that violates some kind of cardinal rule of unit tests.  Essentially it calls the function to test, and calls a local copy-and-paste of the meat of that same function, and expects the results to match, and yes they do.

I'd like to replace this with something, but it needs to work across platforms.  Maybe we need a way to mock the Cesium `writeTextToCanvas` or the 3rd-party `measureText` for the purpose of testing just the screen space calculations.